identity-apps-core/apps/authentication-portal/src/main/webapp/emailOtp.jsp

```diff
@@ -176,7 +176,7 @@
                 <%
                     if ("true".equals(authenticationFailed)) {
                 %>
-                <div class="ui negative message" id="failed-msg"><%=AuthenticationEndpointUtil.i18n(resourceBundle, Encode.forJava(errorMessage))%>
+                <div class="ui negative message" id="failed-msg"><%=Encode.forHtmlContent(errorMessage)%>
```

Comment:
Won't this affect i18N ?, it seems like we have removed the `AuthenticationEndpointUtil.i18n`, is this intentional ?

Reply:
From https://github.com/wso2/identity-apps/blob/6a5cba19182d97b6c0c74d59e81b8851d861c1a6/identity-apps-core/apps/authentication-portal/src/main/webapp/emailOtp.jsp#L80 we are getting the error message from i18n, the issue is with the encoding. that's why use the html encoding as in https://github.com/wso2/identity-apps/blob/e4f8df4ffa3d4ce98effed8345920cc4759e5c01/identity-apps-core/apps/authentication-portal/src/main/webapp/smsOtp.jsp#L141

Reply:
Ahh okay, approving the PR